### PR TITLE
Issue #2: Created devcontainer, requirements.txt

### PR DESCRIPTION
**Pull request by RussellJimmies**
#2

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+	// "forwardPorts": [],
+
+	// Use 'postCreateCommand' to run commands after the container is created.
+	"postCreateCommand": "pip3 install --user -r requirements.txt",
+
+	// Configure tool-specific properties.
+	"customizations": {
+		"vscode": {
+		  "extensions": [
+			"charliermarsh.ruff",
```
Comment (rngadam) [inline .devcontainer/devcontainer.json:24]: this is missing new extensions we've decided to standardize on. 

We should move to our own CFIA AI Lab specific extension pack:

https://code.visualstudio.com/blogs/2017/03/07/extension-pack-roundup

File: .devcontainer/devcontainer.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+			"GitHub.vscode-pull-request-github",
+			"ms-python.python",
+			"ms-python.black-formatter",
+		  ]
+		}
+	  }
+
+	// Uncomment to connect as root instead. More info: https://aka.ms/dev-containers-non-root.
+	// "remoteUser": "root"
+}
```
Comment (rngadam) [inline .devcontainer/devcontainer.json:34]: EOF linefeed

**Review by rngadam — CHANGES_REQUESTED** [review]

File: .devcontainer/devcontainer.json (hunk 1 above)
Comment (RussellJimmies) [reply .devcontainer/devcontainer.json:24]: I have all four extensions listed in the Dev Environment Setup Guide for a python project: https://github.com/ai-cfia/dev-rel-docs/blob/main/Development-Environment-Setup-Guide/DEV-ENV-SETUP.md#default-extensions

File: .devcontainer/devcontainer.json (hunk 2 above)
Comment (RussellJimmies) [reply .devcontainer/devcontainer.json:34]: Done

File: .devcontainer/devcontainer.json (hunk 1 above)
Comment (rngadam) [reply .devcontainer/devcontainer.json:24]: I might be confused, I think the new IDE setup changes were actually configuration changes not new extensions, my bad. 

But I still think we should queue up a new issue for creating our own extension pack so we can easily add extensions across.

**Review by rngadam — APPROVED** [review]
What's unclear to me is how this devcontainer will be used and by who?

**Comment by RussellJimmies** [comment]
I'm about to go develop in this repository

**Pull request merged**